contracts/Colony.sol

```diff
@@ -105,6 +119,26 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return token;
   }
 
+  function emitDomainReputationPenalty(
+    uint256 _permissionDomainId,
+    uint256 _childSkillIndex,
+    uint256 _domainId,
+    address _user,
+    int256 _amount
+  ) public stoppable authDomain(_permissionDomainId, _childSkillIndex, _domainId)
+  {
+    require(_amount <= 0, "colony-penalty-not-negative");
```

Comment:
Fair point. After staring at it for a while, I think the best solution might be to make the input a `uint128` and interpret it as the (positive) reputation penalty, and then cast it to `-int256` before emitting. That would avoid the need for a `require` entirely.

Reply:
That massively changes the valid range of inputs - if the other discussion around `uint128`s pans out, then fine, but for now that's surely not a runner?

Reply:
Erm you are right, I was thinking of `UINT255`, which has the same range as the signed halves of the `INT256` domain (in my mind I thought "half" but it was the wrong kind of half). But that seems less fun since `UINT255` isn't a basic type so let's keep doing what we're doing and change the `require` message.

This applies even if we switch to `UINT128`-size payments since the "half" would be `UINT127` which is still not a thing.